Add Rumble as a third-party project.

ghislainfourny opened the pull request.

ghislainfourny committed:
Add Rumble as a third-party project.

ghislainfourny committed:
Fix URL.

ghislainfourny committed:
Add for Apache Spark in name to clarify.

felixcheung reviewed (approved):
interesting. LGTM (maybe revert the change in site/sitemap.xml)

ghislainfourny committed:
Revert change to site/sitemap.xml by Jekyll.

ghislainfourny commented:
Thanks a lot, Felix! Much appreciated.

I reverted site/sitemap.xml.

Jekyll also modified site/developer-tools.html, let me know if you would like me to revert it as well.

felixcheung commented:
I think that’s fine. Thx
